cmd/duration_test.go

```diff
@@ -294,3 +294,43 @@ func TestDurationUnmarshalText(t *testing.T) {
 		})
 	}
 }
+
+func TestRoundDuration(t *testing.T) {
+	tests := []struct {
+		name     string
+		in       cmd.Duration
+		expected cmd.Duration
+	}{
+		{
+			name:     "15.151 milliseconds",
+			in:       cmd.Duration{15151 * time.Microsecond},
+			expected: cmd.Duration{15 * time.Millisecond},
+		},
+		{
+			name:     "15.151515 milliseconds",
+			in:       cmd.Duration{15151515 * time.Nanosecond},
+			expected: cmd.Duration{15 * time.Millisecond},
+		},
+		{
+			name:     "2.344444 seconds",
+			in:       cmd.Duration{2344444 * time.Microsecond},
+			expected: cmd.Duration{2340 * time.Millisecond},
```

Comment:
why is this rounded? it should be "2344" no?

Reply:
I'm expecting it to be `2.34` seconds. However, time uses only whole numbers, meaning I should test with the lower magnitude, in this case ms. So, to test 2.34s I need 2340ms. If I wanted let's say 2.3s, I would need 2300ms, etc.